src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -374,6 +374,27 @@ public void MaxRequestContentBufferSize_Set_ThrowsPlatformNotSupportedException(
             }
         }
 
+        [Fact]
+        public async Task SendAsync_ReadFromSlowStreamingServer_PartialDataReturned()
+        {
+            // TODO: This is a placeholder until GitHub Issue #2383 gets resolved.
+            const string SlowStreamingServer = "http://httpbin.org/drip?numbytes=8192&duration=15&delay=1&code=200";
```

Comment:
@stephentoub Ok. I ran this single test several times and it did average around 20 seconds on my dev box.

We're currently the httpbin.org server for this "slow streaming" test.  It might have behaviors we don't understand. It is supposed to "drip" the 8192 bytes over 15 seconds at most. 

This is one of the difficulties of getting networking tests of this nature as functional tests over the real internet. For the WinHttpHandler fix itself, I wrote unit tests so that we can mock the network and get exact behaviors we want in order to test the desired codepaths.

I wanted to have a functional test in HttpClientHandler to validate that CurlHandler is behaving correctly.  This particular test is not very different from an actual customer scenario for which this bug was fixed.

If we think this current test is too brittle or unreliable, then I can remove it. But we still need a way of validating CurlHandler for this streaming scenario.